### DISCUSSION TITLE
顶部“搜索框”里的广告如何关闭

### DISCUSSION THREAD

**Discussion by ruanfei** [the1812/Bilibili-Evolved#3418] [提问 / Q&A] [answered]
记得v1有关闭按钮的，刚换v2没找着在哪

**Comment by Geeyun-JY3** [comment, marked as answer]
原版顶栏还是自定义顶栏

***

回顾了一下 [相关功能的变迁](https://github.com/the1812/Bilibili-Evolved/issues/801#issuecomment-1120125067) 后的编辑：

- 如果是原版顶栏，说是可以做：https://github.com/the1812/Bilibili-Evolved/issues/801#issuecomment-1120125654
- 如果是自定义顶栏，把 [搜索栏 - 搜索推荐](https://github.com/the1812/Bilibili-Evolved/blob/master/doc/features/features.md#%E6%90%9C%E7%B4%A2%E6%A0%8F---%E6%90%9C%E7%B4%A2%E6%8E%A8%E8%8D%90) 这个组件卸掉就行了。

**Reply by ruanfei** [reply]
非常感谢
